Symfony/CS/Console/Command/FixCommand.php

```diff
@@ -21,7 +21,10 @@
 use Symfony\CS\Config\Config;
 use Symfony\CS\ConfigInterface;
 use Symfony\CS\Console\ConfigurationResolver;
+use Symfony\CS\Console\Output\JsonOutput;
 use Symfony\CS\Console\Output\ProcessOutput;
+use Symfony\CS\Console\Output\TxtOutput;
+use Symfony\CS\Console\Output\XmlOutput;
```

Comment:
It looks like we have `txt` format, `xml` format, `json` format and... `process` format :D